docs/platform/one-click/deploy-mysql-with-one-click-apps/index.md

```diff
@@ -0,0 +1,154 @@
+keywords: ['database','mysql','rdbms','relational database','mariadb']
+license: '[CC BY-ND 4.0](https://creativecommons.org/licenses/by-nd/4.0)'
+published: 2020-03-13
+modified: 2020-03-13
+modified_by:
+  name: Rajakavitha Kodhandapani
+title: "How to Deploy MySQL/MariaDB with One-Click Apps"
+h1_title: "Deploy MySQL/MariaDB with One-Click Apps"
+contributor:
+  name: Rajakavitha Kodhandapani
+external_resources:
+- '[MySQL 5.6 Reference Manual](https://dev.mysql.com/doc/refman/5.6/en/index.html)'
+- '[PHP MySQL Manual](http://us2.php.net/manual/en/book.mysql.php)'
+- '[MySQLdb User''s Guide](http://mysql-python.sourceforge.net/MySQLdb.html)'
+
+---
+
+## MySQL One-Click App
+
+MySQL is an open-source database management system that uses a relational database and SQL (Structured Query Language) to manage its data. In Debian 9, MySQL is replaced with MariaDB as the default database system. MariaDB is an open-source, multi-threaded relational database management system, backward compatible replacement for MySQL. It is maintained and developed by the MariaDB Foundation.
+
+### Deploy a MySQL/MariaDB One-Click App
+
+{{< content deploy-one-click-apps>}}
+
+### MySQL/MariaDB Options
+
+| **Field** | **Description** |
+|:--------------|:------------|
+| **MySQL User** | The user for your MySQLDB database. *Required*. |
+| **MySQL Root Password** | The root password for your MySQL database. *Required*. |
+| **Create Database** | The database on your MySQL . *Required*. |
+### Linode Options
+
+| **Configuration** | **Description** |
+|:--------------|:------------|
+| **Select an Image** | Debian 9 is currently the only image supported by the MySQL One-Click App, and it is pre-selected on the Linode creation page. *Required*. |
+| **Region** | The region where you would like your Linode to reside. In general, it's best to choose a location that's closest to you. For more information on choosing a DC, review the [How to Choose a Data Center](/docs/platform/how-to-choose-a-data-center) guide. You can also generate [MTR reports](/docs/networking/diagnostics/diagnosing-network-issues-with-mtr/) for a deeper look at the network routes between you and each of our data centers. *Required*. |
+| **Linode Plan** | Your Linode's [hardware resources](/docs/platform/how-to-choose-a-linode-plan/#hardware-resource-definitions). The Linode plan you deploy your MERN stack on should account for the estimated workload. If you are standing up a simple web page, you can use a Nanode or 2GB Linode. If you will deploy a more robust web app, then consider a plan with higher RAM and CPU allocations. If you decide that you need more or fewer hardware resources after you deploy your app, you can always [resize your Linode](/docs/platform/disk-images/resizing-a-linode/) to a different plan. *Required*. |
```

Comment:
Change copy to reflect MySQL app, not MERN stack.

Reply:
fixed.